docs/Concepts/Pruning.md

```diff
@@ -3,11 +3,11 @@ description: Pruning
 
 # Pruning
 
-Use pruning to reduce storage required for the chain. As the number of transactions increases, the world state grows. Pruning removes state tries that aren't required.
+Use pruning to reduce storage required for the world state size. Pruning removes state trie nodes that aren't required.
 
-!!! Caution
-    If you plan to use [private transactions](Privacy/Privacy-Overview.md), do not use pruning.
+!!! Important
+    Using pruning with [private transactions](Privacy/Privacy-Overview.md) is not supported.
 
-A full sync to the Ethereum Mainnet with pruning enabled requires 750GB of disk space. To enable pruning, use the [`--pruning-enabled`](../Reference/CLI/CLI-Syntax.md#pruning-enabled) option.
+To run a full sync to the Ethereum Mainnet with pruning enabled, we recommend approximately 750GB of available disk space per node. To enable pruning, use the [`--pruning-enabled`](../Reference/CLI/CLI-Syntax.md#pruning-enabled) option.
```

Comment:
Was the `per node` added because it's common for our users to have multiple nodes using the same storage? If not, I think it's clearer to remove that caveat.

Reply:
Updated.